fairscale/nn/misc/flatten_params_wrapper.py

```diff
@@ -174,7 +174,7 @@ def __init__(
         self._require_backward_grad_sync = True
         # If optimize_backward_concat == True, used to accumulate the
         # fp32 gradients for the flattened parameters
-        self.fp32_grads = []
+        self.fp32_grads = None
```

Comment:
How about renaming this to something like `fp32_flat_grad` to indicate the type and shape after the change?